discovery-provider/plugins/notifications/src/types/dn.ts

```diff
@@ -181,13 +178,19 @@ export interface ChallengeRow {
   id: string
   starting_block?: number | null
   step_count?: number | null
-  type: ChallengeType
+  type: challengetype
```

Comment:
full lowercase seems out of alignment with convention for TS/JS code. Is this a setting somewhere? Also can we be sure that updating these types hasn't broken anything unrelated?

Reply:
<img width="700" alt="image" src="https://github.com/AudiusProject/audius-protocol/assets/2731362/37443526-2078-4b21-9b5e-73f57547987b">

we have named these kind of poorly in the db. `sql-ts`, the library we use, has these as default to snake_case (which we could change, but i think it is the standard to keep it that way because it's how sql tracks it).


Adding `enumNameCasing: 'pascal'` to the config yields, which is also even wrong
```
export enum Reposttype {
  'track' = 'track',
  'playlist' = 'playlist',
  'album' = 'album',
}
```

I don't know if we can ever be sure about not breaking anything else, but I feel pretty good with all the tests passing and types passing as well

Reply:
pointer: https://github.com/AudiusProject/audius-protocol/blob/1b5474a626e0014f1497bfb6cbb049490b3a435b/packages/sql-ts/src/index.ts#L20